src/oas.md

```diff
@@ -557,6 +557,31 @@ servers:
 
 An object representing a Server Variable for server URL template substitution.
 
+The server URL templating is defined by the following [ABNF](https://tools.ietf.org/html/rfc5234) syntax.
+
+```abnf
+server-url-template            = 1*( literals / template-expression )
+template-expression            = "{" template-expression-param-name "}"
+template-expression-param-name = 1*( %x00-79 / %x7C / %x7E-10FFFF ) ; every UTF8 character except { and }
+
+literals       = 1*( %x21 / %x23-24 / %x26 / %x28-3B / %x3D / %x3F-5B
```

Comment:
@baywet do we want to apply your own errata here https://www.rfc-editor.org/errata/eid6937 ?

Reply:
🤦 I had forgotten about this one... Thanks for pointing it out. Since Ralf made a suggestion, I'll go merge it and resolve this one.